bundles/org.openhab.binding.squeezebox/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -94,10 +94,6 @@
 
 		<properties>
 			<property name="vendor">Logitech</property>
-			<property name="modelId"></property>
-			<property name="name"></property>
-			<property name="uid"></property>
-			<property name="ip"></property>
```

Comment:
Why are these removed, they still seem to be used?

Reply:
There is no reason to declare empty properties. They will be set programatically, and until then they were previously shown as empty.